code/game/objects/structures/gate.dm

```diff
@@ -19,7 +19,19 @@ GLOBAL_LIST_EMPTY(biggates)
 	var/gid
 	attacked_sound = list('sound/combat/hits/onmetal/sheet (1).ogg', 'sound/combat/hits/onmetal/sheet (2).ogg')
 	var/obj/structure/attached_to
-
+	/// this is dumb but I'm not refactoring this right meow.
+	var/is_big_gate = TRUE
+	/// which bodyparts we affect when crushing a carbon mob.
+	var/static/list/bodyparts_to_crush = list(
+			BODY_ZONE_HEAD,
+			BODY_ZONE_CHEST,
+			BODY_ZONE_L_ARM,
+			BODY_ZONE_R_ARM,
+			BODY_ZONE_L_LEG,
+			BODY_ZONE_R_LEG,
+		)
+	/// damage multiplier for crushing
+	var/crush_damage_multiplier = 1
```

Comment:
is this necesary ? since the parent just gibs you it doesnt really make sense to have a multiplier.